app/controllers/authy_controller.rb

```diff
@@ -0,0 +1,42 @@
+class AuthyController < ApplicationController
+  protect_from_forgery except: [:callback, :send_token]
+
+  # The webhook setup for our Authy application this is where
+  # the response from a OneTouch request will come
+  def callback
```

Comment:
Good question. Definitely worth looking into. Doesn't feel like a blocker to me.